### ISSUE TITLE
How to train Mask R-CNN for cityscapes?

### ISSUE THREAD

**Issue by hoanganhtuanfsoft** [NVIDIA/vid2vid#103]
as @forfun1012 metioned [https://github.com/NVIDIA/vid2vid/issues/61](here), we use Mask R-CNN to get instance maps. But @tcwang0509 said "This is actually not very important, as the network wasn't trained on Cityscapes and doesn't perform really well."
So I want to train Mask R-CNN for cityscapes again to get results better. But I don't have instance's dataset (growth truth)

Help me by guiding me how to generate the above dataset!

Thank you!

**Comment by BmlmnnsDev** [comment]
1. Register & Download the cityscapes dataset from https://www.cityscapes-dataset.com/
2. Use the cityscapes to coco converter: https://github.com/facebookresearch/Detectron/blob/master/tools/convert_cityscapes_to_coco.py
3. Start from a pretrained Mask R-CNN network https://github.com/matterport/Mask_RCNN#installation and finetune on cityscapes in coco format
